services/core-web/src/components/mine/Compliance/ComplianceOrdersTable.js

```diff
@@ -67,70 +55,33 @@ const columns = [
     dataIndex: "overdue",
     render: (text, record) => (
       <div title="Overdue">
-        {record.overdue && record.due_date !== null ? (
+        {text && record.due_date !== null ? (
           <img className="padding-sm" src={RED_CLOCK} alt="Overdue Report" />
         ) : (
           ""
         )}
       </div>
     ),
   },
-  {
-    title: "Order No.",
-    key: "order_no",
-    dataIndex: "order_no",
-    render: (text, record) => <div title="Order No.">{record.order_no || "-"}</div>,
-    sorter: (a, b) => (a.order_no > b.order_no ? -1 : 1),
-  },
+  renderTextColumn("order_no", "Order No.", true, "-"),
   {
     title: "Violation",
     key: "violation",
     dataIndex: "violation",
-    render: (text, record) => <div title="Violation">{record.violation || "-"}</div>,
+    render: (text) => <div title="Violation">{text || "-"}</div>,
```

Comment:
this was something I wasn't sure about- in other places where there's no cell data, we use "N/A" or just blank. Here we are using "-" which we do nowhere else.

Reply:
That is odd.  Maybe a question for the design team, although I suspect they will want us to make it consistent with the rest of the site.

Reply:
I listed it on the Jira ticket as an issue I didn't deal with. I figure "make empty table cells consistent" can be its own ticket. Starting point would just be setting those "placeHolder" params on column functions = "N/A" (or whatever UX wants)